tools/mt2mq/MT2MQ.xml

```diff
@@ -1,18 +1,19 @@
-<tool id="mt2mq" name="MT2MQ" version="1.0">
+<tool id="mt2mq" name="MT2MQ" version="1.1.0">
     <description>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</description>
     <requirements>
         <requirement type="package" version="1.2.1">r-tidyverse</requirement>
+        <requirement type="package" version="0.9.97">r-taxize</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
 
 #if $options.mode == "f" or $options.mode == "ft":
-    Rscript '$__tool_directory__/MT2MQ.R' '$options.input_files' '$options.mode' $options.ontology '$mq_output'
+    Rscript '$__tool_directory__/MT2MQ.R' '$options.input_files' '$options.mode' '$options.ontology' '$int_output' '$func_output' '$tax_output'
```

Comment:
can you add `--vanilla` ?

Reply:
Done!! Thanks!!

Reply:
@bgruening @jj-umn @bernt-matthias @mariecrane 
Hi, Can this update be merged? Required changes were made by @mariecrane and I have added --vanilla as requested. All the tests seem to have passed successfully. Thanks!